roles/fio-distributed/templates/configmap.yml.j2

```diff
@@ -5,6 +5,26 @@ metadata:
   name: fio-test-{{ trunc_uuid }}
   namespace: '{{ operator_namespace }}'
 data:
+{% if fiod.rook_ceph.prefill is sameas true %}
```

Comment:
Modify conditional for something like this to avoid breaking when the variable is not set
```
{% if prefill is defined and prefill is sameas true %}
```

Reply:
ack. will make change